gocourse09/webinar/main.go

```diff
@@ -0,0 +1,23 @@
+/*
+Написати програму для електронної крапельниці, яка сама має доступ до кількох джерел ліків і сама їх додає/змішує
+взалежності від стану пацієнта.
+Притримуванися SOLID
+*/
+package main
+
+import (
+	"github.com/sarff/prjctr-golang_beginning/gocourse09/webinar/dropper"
+	"github.com/sarff/prjctr-golang_beginning/gocourse09/webinar/drug"
+	"github.com/sarff/prjctr-golang_beginning/gocourse09/webinar/patient"
+)
+
+func main() {
+	patient1 := &patient.Patient{}
```

Comment:
> не можемо назвати patient так як є імпортований модуль

Можемо, спробуй:

```suggestion
	patient := &patient.Patient{}
```

Програма компілюється та працює.

В кожного пацієнта є ім'я. Можна на ім'я його назвати:

```go
john := &patient.Patient{}
```

Reply:
можемо, але маю ось таке попередження 
<img width="533" alt="image" src="https://github.com/user-attachments/assets/3e801b9e-3950-47b4-9910-cd5272dc2a4b">